packages/flutter/test/widgets/windowing_test.dart

```diff
@@ -110,23 +111,22 @@ void main() {
         isWindowingEnabled = true;
       });
 
-      test('createDefaultOwner returns a WindowingOwner', () {
-        final WindowingOwner owner = WindowingOwner.createDefaultOwner();
-        expect(owner, isA<WindowingOwner>());
-      });
-
       testWidgets('RegularWindow does not throw', (WidgetTester tester) async {
+        final _StubWindowController controller = _StubWindowController(tester);
         await tester.pumpWidget(
           wrapWithView: false,
-          RegularWindow(controller: _StubWindowController(tester), child: Container()),
+          RegularWindow(controller: controller, child: Container()),
         );
+
+        addTearDown(controller.dispose);
```

Comment:
Nit: I'd move the `addTearDown` to be directly after the line that creates the controller. Same comment for other lines below.